Server/helpers/Validation.js

```diff
@@ -29,4 +29,27 @@ const validateLoginInfo = (userData) => {
   return schema.validate(userData);
 };
 
-module.exports = { validateEmail, validateLoginInfo };
+const validatesignupInfo = (userData) => {
+  const schema = joi.object({
+    email: joi
+      .string()
+      .min(6)
+      .required()
+      .email(),
+    userName: joi
+      .string()
+      .min(5)
+      .max(30)
+      .required()
+      .pattern(new RegExp('^[a-zA-Z0-9_]')),
```

Comment:
the pattern you used for username tells the user that the name should contain characters and numbers.